templates/vue/src/App.vue

```diff
@@ -38,7 +52,26 @@ export default {
       return Object.keys(this.nodes).length === 0
     },
   },
+  watch: {
+    loggedIn(newValue) {
```

Comment:
Instead of `newValue` can we call this something more meaningful like `isStillLoggedIn`